packages/bpk-component-banner-alert/src/BpkBannerAlert-test.js

```diff
@@ -78,15 +80,30 @@ describe('BpkBannerAlert', () => {
 
   it('should render correctly with dismissable option', () => {
     const tree = renderer.create(
-      <BpkBannerAlert type={ALERT_TYPES.WARN} message={message} dismissable />,
+      <BpkBannerAlert type={ALERT_TYPES.WARN} message={message} dismissable dismissButtonLabel="Dismiss" />,
```

Comment:
I think it makes sense to include the label in the snapshot

Reply:
Sure! Good catch 😸